backend/vcdat/htmlcov/Files_py.html

```diff
@@ -0,0 +1,181 @@
+
```

Comment:
What is this file, and why is it needed? same goes for all the htmlcov files? if these are generated coverage output, They don't need to be in the repo. update the .gitignore file `backend/vcdat/htmlcov`